components/eamxx/src/physics/share/scream_trcmix.hpp

```diff
@@ -16,8 +16,10 @@ using trcmix_view1d = typename ekat::KokkosTypes<DefaultDevice>::template view_1
 template<typename Scalar>
 using trcmix_view2d = typename ekat::KokkosTypes<DefaultDevice>::template view_2d<Scalar>;
 
+// NOTE: nlevs be smaller than pmid/q extent, in case one/both of them are padded
```

Comment:
Typo: nlevs *can* be smaller

Reply:
To avoid another round of testing and approval (in order to merge with all checks passed), I'll fix it in my next PR.